kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
+        maxfiles = Config.getint('kivy', 'log_maxfiles') - 1
```

Comment:
I'm not sure if `- 1` is a good idea as the rest of the code expects `100` and not `99` as a value for the current behavior (`if len(lst) > maxfiles:`, `lst = lst[:-maxfiles]`, etc) . Probably just a matter of taste though.